sdk/keyvault-v2/azure-security-keyvault-administration/src/main/java/com/azure/v2/security/keyvault/administration/KeyVaultAccessControlClient.java

```diff
@@ -279,18 +278,12 @@ public PagedIterable<KeyVaultRoleDefinition> listRoleDefinitions(KeyVaultRoleSco
     public PagedIterable<KeyVaultRoleDefinition> listRoleDefinitions(KeyVaultRoleScope roleScope,
         RequestContext requestContext) {
 
-        try {
-            Objects.requireNonNull(roleScope, String.format(CANNOT_BE_NULL, "'roleScope'"));
-
-            return mapPages(
-                pagingOptions -> clientImpl.getRoleDefinitions()
-                    .listSinglePage(roleScope.getValue(), null, requestContext),
-                (pagingOptions, nextLink) -> clientImpl.getRoleDefinitions()
-                    .listNextSinglePage(nextLink, requestContext),
-                KeyVaultAdministrationUtil::roleDefinitionToKeyVaultRoleDefinition);
-        } catch (RuntimeException e) {
-            throw LOGGER.logThrowableAsError(e);
-        }
+        Objects.requireNonNull(roleScope, "'roleScope' cannot be null.");
```

Comment:
the `String.format(CANNOT_BE_NULL, "'roleScope'")` happens on every operation and there is really no need to do it